### PR TITLE
minor changes in the documentation in general 

### PR DESCRIPTION
**Pull request by RafaelPignata622**
I add the motivation into the readme
change three for six methods 
and modificate the API introduction

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,13 @@
 
 It runs in numerical N-body simulations populated with Semi-analytical models and full Hydrodynamical simulations, such as Illustris TNG.
 
+## Motivation
+
+ Galaxies are self-gravitating complex stellar systems formedmainly by stars, dark matter, gas and dust. Star are assembledin different stellar components, such as the disk (thin and thick),the nucleus, the stellar halo and the bar. The components interact with each other and each of them follows its own temporal evolution. For this reason, the description of the formation and evolution  of  galaxies  is  strongly  linked  to  the  formation  and evolution of each of these individual components and their assembly into the final galaxy.
```
Comment (nvillagra) [inline README.md:15]: ...formed mainly... 
...assemble in...

**Review by vcristiani — APPROVED** [review]

**Review by antotaverna — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -10,6 +10,13 @@
...
+ 
+ A dynamical descomposition it is a fundamental tool to be able to separate each components of the galaxies for further study. Numerous methods exist in the literature to perform this task, but there is no tool that allows us to use several of them, providing the possibility of an easy comparison.
+
+
 ## Dynamic decomposition method implemented
```
Comment (antotaverna) [inline README.md:20]: Model ??

**Review by nvillagra — APPROVED** [review]
Everything looks fine, except for two tiny typing problems.

**Comment by antotaverna** [comment]
Joia

**Pull request merged**